src/main.ts

```diff
@@ -220,12 +225,12 @@ const createWindow = async () => {
 
 app.whenReady().then(
   async () => {
+    await createSplashWindow()
```

Comment:
This isn't related to the issue I'm trying to fix, but I figured it should come first.